### DISCUSSION TITLE
J’ai testé la garantie à Vie Corsair - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#48] [Comments]
# J’ai testé la garantie à Vie Corsair - Haade.fr

j’ai testé la prise en charge de la garantie à vie des mémoires corsair ddr4 vengeance, retour d’expérience

https://haade.fr/fr/blog/memoire-corsair-ddr4-garantie-j-ai-teste

**Comment by membrelyrique7** [comment]
Bonjour, 
dans l'article il est indiqué que le test memtest86+ fourni un lien vers les résultats mais ou se trouve ce lien ? dans le paramétrage  du test je ne vois pas d'option d'export des résultats 😕
